File: Modul_4/Bahasa_Python/PRAK403-2510817120005-NorKhalisyah.py
a, b= map(int, input("Masukkan dua Angka: ").split())

kiri = a
kanan = b

if a < b:
    while kiri <= b and kanan >= a:
        print(kiri, kanan, end="")
        kiri += 1
        kanan -= 1
        if kiri <= b and kanan >= a:
            print(" - ", end="")

else:
    while kiri >= b and kanan <= a:
        print(kiri, kanan, end="")
        kiri -= 1
        kanan += 1
        if kiri >= b and kanan <= a:
            print(" - ", end="")

File: Modul_4/Bahasa_C/PRAK403-2510817120005-NorKhalisyah.c
#include <stdio.h>
int main() {

    int a, b;
    printf("Masukkan dua angka: ");
    scanf("%d %d", &a, &b);

    int kiri = a, kanan = b;

    if (a < b) {
        while (kiri <= b&& kanan >=a) {
            printf("%d %d", kiri, kanan);
            kiri++;
            kanan--;
            if (kiri <= b && kanan >=a)
                printf(" - ");
        }
    }

    else {
        while (kiri >= b && kanan <= a){
            printf("%d %d", kiri, kanan);
            kiri--;
            kanan++;
            if (kiri >= b && kanan <= a)
                printf(" - ");
        }
    }

    return 0;
}
